frontend/src/scenes/feature-flags/FeatureFlagProjects.tsx

```diff
@@ -0,0 +1,115 @@
+import { LemonTable, LemonTableColumns } from 'lib/lemon-ui/LemonTable'
+import { LemonButton, LemonModal, LemonModalProps, LemonSelect } from '@posthog/lemon-ui'
+import { IconArrowRight } from 'lib/lemon-ui/icons'
+import { useActions, useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { Field } from 'lib/forms/Field'
+import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
+import { featureFlagLogic } from './featureFlagLogic'
+import { organizationLogic } from '../organizationLogic'
+
+const records: Record<string, string>[] = [{ project: 'project-1-staging', flag_status: 'enabled' }]
+
+export function CopyFeatureFlagModal({
+    overlayRef,
+    contentRef,
+}: Pick<LemonModalProps, 'overlayRef' | 'contentRef'>): JSX.Element {
+    const { featureFlag, isProjectsModalOpen } = useValues(featureFlagLogic)
+    const { currentOrganization } = useValues(organizationLogic)
+    const { closeProjectsModal } = useActions(featureFlagLogic)
+
+    return (
+        <LemonModal
+            overlayRef={overlayRef}
+            contentRef={contentRef}
+            isOpen={isProjectsModalOpen}
+            onClose={closeProjectsModal}
+            title="Feature Flag copy"
+            description="Copy your flag and its configuration to another project"
+            footer={
+                <div className="flex-1 flex items-center justify-between">
+                    <div className="flex items-center gap-2" />
+                    <div className="flex items-center gap-2">
+                        <LemonButton form="ff-copy-modal-form" type="secondary" onClick={closeProjectsModal}>
+                            Cancel
+                        </LemonButton>
+                        <LemonButton htmlType="submit" type="primary">
+                            Copy
+                        </LemonButton>
+                    </div>
+                </div>
+            }
+        >
+            <div className="max-w-120">
+                <Form
+                    logic={featureFlagLogic}
```

Comment:
depending on whether there's a bindlogic before the tabs or not, this might require the props to be passed in  👀